radio/src/targets/horus/CMakeLists.txt

```diff
@@ -1,7 +1,6 @@
 option(DISK_CACHE "Enable SD card disk cache" ON)
 option(UNEXPECTED_SHUTDOWN "Enable the Unexpected Shutdown screen" ON)
 option(FLYSKY_HALL_STICKS "Enable FlySky Hall Sticks" OFF)
-option(IMU_LSM6DS33 "Enable I2C2 and LSM6DS33 IMU" OFF)
```

Comment:
I think you need to rebase or merge the latest state of `main` into your branch, because this and the two references to IMU_LSM6DS33 were merged 4 days ago. ;)

Reply:
how do i do that?  do i do a pull and then reissue the pull request?

Reply:
Have a look at the following link:
https://github.com/EdgeTX/edgetx/pull/285#issuecomment-883332611

Reply:
With a rebase - you will need to do a forced push after getting it up to date again. With a merge, it will add another commit, and will push normally, and update the PR to suit (since the PR is a reflection of the state of the branch the PR is based on). rebase is cleaner for commit history, but can be a pain if it is the first time you've done on.